src/legacy/core_plugins/kibana/public/visualize/wizard/_dialog.scss

```diff
@@ -5,6 +5,11 @@
   background-size: 37%;
 }
 
+.visNewVisSearchDialog {
+  min-height: 480px;
```

Comment:
Can you use SASS variables instead of `px` values? For instance this should be `$euiSizeL * 20`